hoomd/trigger.py

```diff
@@ -1,14 +1,58 @@
 # Copyright (c) 2009-2019 The Regents of the University of Michigan
-# This file is part of the HOOMD-blue project, released under the BSD 3-Clause
 # License.
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause
```

Comment:
Related: @joaander this could be less than 80 characters and fit on one line if we replace "the HOOMD-blue project" with "HOOMD-blue." I don't think there is any legal/other reason to prefer the longer form.
```python
# This file is part of HOOMD-blue, released under the BSD 3-Clause License.
```